components/automate-chef-io/content/docs/iam-v2-api-reference.md

```diff
@@ -10,60 +10,145 @@ toc = true
+   Use this, filling in a token name of your choice:
 
    ```bash
-   export TOKEN=`chef-automate iam token create TEST_ADMIN --admin`
+   export TOKEN=`chef-automate iam token create <your-token-name-here> --admin`
    ```
 
-## Policies
+2. URIs are relative to `https://<your-domain-here>/apis/iam/v2beta`, unless otherwise noted.
 
-### Listing Policies
+3. Attach the `?pretty` query string to an endpoint to get pretty-printed output.
+
+Putting it all together with the `/policies` endpoint as an example, this command fetches the list of policies with multi-line, formatted output:
 
 ```bash
-curl -s -H "api-token: $TOKEN" https://{{< example_fqdn "automate" >}}/apis/iam/v2beta/policies?pretty
+curl -sH "api-token: $TOKEN" \
+  https://{{< example_fqdn "automate" >}}/apis/iam/v2beta/policies?pretty
 ```
 
-The output includes both `Chef-managed` and `custom` policies.
-Policies migrated from IAM v1 are included as `custom` policies.
-Policy names that begin with `[Legacy]` are default policies migrated from v1.
-We recommend deleting these legacy policies and setting up new IAM v2 policies in their place.
-
-In the browser: Navigate to the *Policies* page under the **Settings**  tab.
+For those API methods that take JSON data (typically `create` and `update` methods)
+you can provide that data to the REST endpoint in a variety of ways.
+If it is a relatively small size payload, you can include it in a `curl` command inline, e.g.
 
-### Getting a Policy
+```bash
+curl -sH "api-token: $TOKEN" -d '<your JSON here>' ...
+```
 
-In order to fetch a policy on the command-line, you must know its ID.
-This may be obtained by first fetching the list of all policies; see [Listing Policies]({{< relref "iam-v2-api-reference.md#listing-policies" >}}).
-Find the `id` field for the policy of interest--for example, the ID from the default administrator policy is `administrator-access`.
-Plug that in to the URL as shown:
+If the payload is larger, it is often convenient to store it in a file, then pass that file, e.g.,
 
 ```bash
-curl -s -H "api-token: $TOKEN" https://{{< example_fqdn "automate" >}}/apis/iam/v2beta/policies/administrator-access?pretty
+curl -sH "api-token: $TOKEN" -d @policy.json ...
 ```
 
-In the browser: Navigate to the *Policies* page under the **Settings**  tab, and then select an individual policy to open its details.
+## Notes on the Projects Property
 
-### Creating a Policy
+Teams, tokens, roles, and policies all contain a top-level `projects` property; you will see that in the example JSON for each resource in the following sections.
+This property associates the particular IAM resource with one or more projects.
+Users must have permissions for those projects to be able to view or modify those resources.
+This `projects` property may be empty, which would give access only to users included on policies specifying the special `(unassigned)` project designation.
```

Comment:
I think
> the special `(unassigned)` project designation 
is bleeding some implementation detail. Maybe 
`If the projects property is left empty, that resource will be categorized as `unassigned`.  ?